src/actions/index.js

```diff
@@ -15,35 +14,68 @@ export function receiveLocation(newLocation) {
   };
 }
 
+export function displaySuccess() {
+  return {
+    type: DISPLAY_SUCCESS,
+    message: 'Request succeeded',
+    color: 'green',
```

Comment:
I would advise storing this as an abstract attribute rather than making aesthetic choices in your data store. It would be preferable to store `request_status` or similar and then use that as a className to attach the color attributes to. TL;DR - separate appearance from logic